rosdep/python.yaml

```diff
@@ -4706,6 +4706,9 @@ python-tk:
   arch: [python2, tk]
   debian: [python-tk]
   fedora: [python2-tkinter]
+  rhel:
+    '*': [python2-tkinter]
```

Comment:
Not sure if useful; pasting links to some mirror/index of RPMs called rpmfind
python2-tkinter in CentOS 8 http://rpmfind.net/linux/RPM/centos/8-stream/appstream/aarch64/Packages/python2-tkinter-2.7.16-12.module_el8.1.0+219+cf9e6ac9.aarch64.html